content/en/blog/2024/v1-apis/index.md

```diff
@@ -0,0 +1,81 @@
+---
+title: "Getting Started with using v1 Istio APIs "
+description: Announcing the v1 Networking, Security and Telemetry APIs. 
+publishdate: 2024-05-13
+attribution: Whitney Griffith - Microsoft
+keywords: [istio, traffic, security, telemetry, API]
+target_release: 1.22
+---
+
+Istio has several [Networking](/docs/reference/config/networking/), [Security](/docs/reference/config/security/) and [Telemetry](/docs/reference/config/telemetry/) APIs that are crucial for ensuring the secure, seamless connectivity, and effective monitoring of services within the Service Mesh layer. These APIs are used on thousands of clusters across the world, securing and enhancing critical infrastructure. As a reflection of the stability, adoption, and value of these resources, the Istio community has decided to promote many of our CRDs to `v1` in Istio 1.22.
+
+In Istio 1.22 we are happy to announce that a concerted effort has been made to graduate the below APIs to `v1`:
+* [Destination Rule](/docs/reference/config/networking/destination-rule/)
+* [Gateway](/docs/reference/config/networking/gateway/)
+* [Service Entry](/docs/reference/config/networking/service-entry/)
+* [Sidecar](/docs/reference/config/networking/sidecar/)
+* [Virtual Service](/docs/reference/config/networking/virtual-service/)
+* [Workload Entry](/docs/reference/config/networking/workload-entry/)
+* [Workload Group](/docs/reference/config/networking/workload-group/)
+* [Telemetry API](/docs/reference/config/telemetry/)*
+* [Peer Authentication](/docs/reference/config/security/peer_authentication/)
+
+## Telemetry API
+
+The `v1` Telemetry API is the only API that was promoted that had changes from its previous version. The following features weren’t promoted to `v1`:
+* `metrics.reportingInterval`
+    * Reporting interval allows configuration of the time between calls out to for metrics reporting. This currently only supports TCP metrics but we may use this for long duration HTTP streams in the future.
+    * At this time, Istio lacks usage data to support the need for this feature.
+* `accessLogging.filter`
+    * If specified, this filter will be used to select specific requests/connections for logging.
+    * This feature is based on a relatively new feature in Envoy, and Istio needs to further develop the use case and implementation before graduating it to `v1`.
+* `tracing.useRequestIdForTraceSampling`
+    * This value is true by default. The format of this Request ID is specific to Envoy, and if the Request ID generated by the proxy that receives user traffic first is not specific to Envoy, Envoy will break the trace because it cannot interpret the Request ID. By setting this value to false, we can prevent [Envoy from sampling based on the Request ID](https://www.envoyproxy.io/docs/envoy/latest/intro/arch_overview/observability/tracing#trace-context-propagation).
+    * There is not a strong use case for making this configurable through the Telemetry API.
```

Comment:
It would be great to have a couple of sentences here telling users to provide feedback via GitHub if they feel strongly about this fields

Reply:
Added!